File: java/pe/src/test/java/ktn/pe/Pe11Test.java
package ktn.pe;

import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.*;

import org.junit.Before;
import org.junit.Test;

public class Pe11Test {
    private final Pe11 pe11 = new Pe11();

    @Before
    public void setUp() {
        pe11.setFname("../../res/pe11.txt");
    }

    @Test
    public void test() {
        assertThat("70600674", pe11.pe11(), is(70600674));
    }

}

File: kt/pe/src/test/kotlin/ktn/pe/Pe11Test.kt
package ktn.pe

import org.hamcrest.CoreMatchers.`is`
import org.junit.Assert.*

import org.junit.Before
import org.junit.Test

class Pe11Test {
    private val pe11 = Pe11()

    @Before
    @Throws(Exception::class)
    fun setUp() {
        pe11.fname = "../../res/pe11.txt"
    }

    @Test
    fun test() {
        assertThat("70600674", pe11.pe11(), `is`(70600674))
    }

}
